.gitignore

```diff
@@ -1,4 +1,3 @@
 node_modules
 .DS_Store
 .nyc_output
-dist
```

Comment:
This can be kept since a build step is handled by the [`publish.yaml`](https://github.com/slackapi/slack-github-action/blob/main/.github/workflows/publish.yaml#L16) workflow